src/Servers/IIS/IIS/test/Common.FunctionalTests/Inprocess/StartupTests.cs

```diff
@@ -880,6 +880,54 @@ public async Task StackOverflowCanBeSetBySettingLargerStackViaHandlerSetting()
             Assert.True(result.IsSuccessStatusCode);
         }
 
+        [ConditionalTheory]
+        [RequiresIIS(IISCapability.PoolEnvironmentVariables)]
+        [RequiresNewShim]
+        [RequiresNewHandler]
+        [InlineData(HostingModel.InProcess)]
+        [InlineData(HostingModel.OutOfProcess)]
+        public async Task EnvironmentVariableForLauncherPathIsPreferred(HostingModel hostingModel)
+        {
+            var deploymentParameters = Fixture.GetBaseDeploymentParameters(hostingModel);
+
+            deploymentParameters.EnvironmentVariables["ANCM_LAUNCHER_PATH"] = "nope";
```

Comment:
@jkotalik I'm hoping you might have some ideas of how to write some positive tests instead of just negative 😄

Reply:
You could copy the app to a different temporary folder and set ANCM_LAUNCHER_PATH/LAUNCHER_ARGS to that folder?

Reply:
Shouldn't need to copy anything, but how do I know where the app is located? And is launcher path always just dotnet.exe? Because I have `_dotnetLocation` available, but I don't know what to give launcher args.